broker-daemon/broker-rpc/wallet-service/commit.js

```diff
@@ -112,8 +134,10 @@ async function commit ({ params, relayer, logger, engines, orderbooks }, { Empty
     logger.debug('Requesting inbound channel from relayer', { address: paymentChannelNetworkAddress, balance: convertBalance, symbol: inverseSymbol })
     await relayer.paymentChannelNetworkService.createChannel({address: paymentChannelNetworkAddress, balance: convertedBalance, symbol: inverseSymbol})
   } catch (e) {
-    // TODO: Close channel that was open if relayer call has failed
-    throw (e)
+    // If the relayer call fails, the user can simply try and open channels again
+    // on the same market. We handle this undesired state above where we check the
+    // current status of channels and repair them if necessary.
+    throw new PublicError(`Funding error: Relayer is unavailable, Please try again`)
```

Comment:
not sure that `Relayer is unavailable` is the message I would send here. At the very least we should log the actual error, and given an error like `Relayer error`.